frontends/web/src/routes/buy/pocket.tsx

```diff
@@ -152,6 +102,113 @@ export const Pocket = ({ code }: TProps) => {
     current.contentWindow?.postMessage(message, '*');
   };
 
+  const handleRequestAddress = (message: RequestAddressV0Message) => {
+    signing = true;
+    const addressType = message.withScriptType ? String(message.withScriptType) : '';
+    const withMessageSignature = message.withMessageSignature ? String(message.withMessageSignature) : '';
```

Comment:
`withMessageSignature` should either be falsy (undefined, null or false) or a string. So I think `String()` is not needed 🤔